actions/service_test.go

```diff
@@ -155,4 +161,10 @@ hooks:
 	if runResult != nil {
 		t.Errorf("GetRunResult() result=%v, expected nil", runResult)
 	}
+
+	bytes, err := ioutil.ReadAll(outputReader)
```

Comment:
need to capture this one in the DoAndReturn - can't assume the reader will be valid at this point